.breakage/get_jso_users.jl

```diff
@@ -0,0 +1,18 @@
+import GitHub, PkgDeps  # both export users()
+
+length(ARGS) >= 1 || error("specify at least one JSO package as argument")
+
+jso_repos, _ = GitHub.repos("JuliaSmoothOptimizers")
+jso_names = [splitext(x.name)[1] for x ∈ jso_repos]
+
+name = splitext(ARGS[1])[1]
+name ∈ jso_names || error("argument should be one of ", jso_names)
+
+dependents = String[]
+try
+    global dependents = filter(x -> x ∈ jso_names, PkgDeps.users(name))
```

Comment:
I think it would be great if there was some mechanism to manually add external packages, maybe by reading an additional file. What do you think?

Reply:
Do you mean that you want to test packages outside of JSO?

Reply:
Is it really our job to test external packages? They could easily test themselves when they bump to a new release of one of our packages.